cocotb/triggers.py

```diff
@@ -127,6 +156,11 @@ def __await__(self):
         """))
 
 
+class Trigger(TriggerBase, TriggerHelper):
+    def __init__(self):
+        TriggerBase.__init__(self)
```

Comment:
No need to define `__init__` here

Reply:
Actually I removed ``TriggerBase`` again and now use:
```python
class Trigger(_py_compat.abc_ABC, TriggerHelper)
```